packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -186,6 +223,17 @@ const Updates = decorate([
           .sort()
           .map(name => `- ${name}: ${installedPackages[name]}`)
           .join('\n'),
+      xoaReleaseChannelsOptions: ({ xoaReleaseChannels }) => [
+        ...map(xoaReleaseChannels, (_, channel) => ({
+          label: channel,
+          value: channel,
+        })),
+        {
+          label: <span className='font-italic'>unlisted channel</span>,
+          value: 'unlisted channel',
+        },
+      ],
+      idReleaseChannelsForm: generateId,
```

Comment:
IMHO, `releaseChannelsFormId` would be better.

(Keep the computed sorted)

Reply:
You can even use `channels` instead of `releaseChannels` in your code if you prefer.